index.html

```diff
@@ -76,7 +76,7 @@
   </head>
   <body>
     <section id="abstract">
-      <p>This specification does neat stuff.</p>
+      <p>This specification defines a JSON Web Token (JWT) [RFC7519] representation of Verifiable Credentials.</p>
```

Comment:
I agree with @TallTed 's suggestion
```suggestion
      <p>
This specification defines how Verifiable Credentials can be represented as JSON
Web Tokens (JWT) [RFC7519] and secured using JSON Web Signatures (JWS)
[RFC7515].
      </p>
```